src/main/java/graphql/schema/SchemaTraverser.java

```diff
@@ -66,9 +66,11 @@ public TraverserResult depthFirstFullSchema(List<GraphQLTypeVisitor> typeVisitor
         roots.addAll(schema.getAdditionalTypes());
         roots.addAll(schema.getDirectives());
         roots.addAll(schema.getSchemaDirectives());
+        roots.addAll(schema.getSchemaAppliedDirectives());
         roots.add(schema.getIntrospectionSchemaType());
         TraverserDelegateListVisitor traverserDelegateListVisitor = new TraverserDelegateListVisitor(typeVisitors);
-        return initTraverser().rootVars(rootVars).traverse(roots, traverserDelegateListVisitor);
+        Traverser<GraphQLSchemaElement> traverser = initTraverser().rootVars(rootVars).rootVar(GraphQLSchema.class, schema);
+        return traverser.traverse(roots, traverserDelegateListVisitor);
```

Comment:
Same here on SchemaTraverser - the schema being traversed is now a root variable